src/tools/general.js

```diff
@@ -64,23 +64,35 @@ async function resolveInBatches(promiseArray, batchLength = 10) {
 }
 
 const processReview = (review, remoteId) => {
-    const { text, title, rating, tripInfo, publishedDate, userProfile } = review;
+    const { text, title, rating, tripInfo, publishedDate, userProfile, photos } = review;
     const stayDate = tripInfo ? tripInfo.stayDate : null;
     let userLocation = null;
     let userContributions = null;
+    let userName = null;
+    let imageUrls = null;
```

Comment:
should be `[]` so it's consistent to the output

Reply:
Done